template_diplomarbeit/template_diplomarbeit.tex

```diff
@@ -21,18 +25,23 @@
 
 \input{kapitel/danksagung}
 
+\newpage
+
+\pagenumbering{arabic}
```

Comment:
Oh, die Seitennummerierung schint durchgängig arabisch zu sein?

Das wäre gut https://github.com/latextemplates/scientific-thesis-template/issues/69

Reply:
die ersten seiten sind aktuell roemisch numeriert, der rest arabisch. in der alten version wurden die letzten seiten wieder roemisch numeriert, was ich fuer am unueblichsten halte und rausgeworfen habe.
ob am anfang roemisch numeriert werden soll, haengt von mehreren sachen ab (deutsch vs. english, vorgaben der uni, geschmackssache des supervisors/profs). da es leichter ist, das roemische zeug rauszuschmeissen als es einzubauen, habe ich es dringelassen.